src/main/java/org/jitsi/videobridge/ice/IceTransport.java

```diff
@@ -453,6 +464,14 @@ private int addRemoteCandidates(
             if ((relAddr = candidate.getRelAddr()) != null
                     && (relPort = candidate.getRelPort()) != -1)
             {
+                boolean relAddrNeedsResolution
+                        = !IPAddressUtil.isIPv4LiteralAddress(relAddr)
+                            && !IPAddressUtil.isIPv6LiteralAddress(relAddr);
+                if (relAddrNeedsResolution && !Config.resolveRemoteCandidates())
+                {
+                    logger.debug(() -> "Ignoring remote candidate with non-literal related address: " + address);
+                    continue;
```

Comment:
I'm not sure it makes sense to completely ignore a candidate with a non-literal related address, as opposed to just passing null for that parameter?  I'd be surprised if we use remote candidates' related addresses at all.

(That said, I'd also be surprised to have a candidate with an IP literal for the candidate, and a hostname for the related, so this probably doesn't matter so much.)